docs/nginx-ingress-controllers.md

```diff
@@ -26,7 +26,7 @@ The table below summarizes the key difference between nginxinc/kubernetes-ingres
 | JWT validation | Not supported | Not supported | Supported |
 | Session persistence | Supported via a third-party module | Not supported | Supported |
 | Canary testing (by header, cookie, weight) | Supported via annotations | Supported via custom resources | Supported via custom resources |
-| Configuration templates *1 | See the [template](https://github.com/kubernetes/ingress-nginx/blob/master/rootfs/etc/nginx/template/nginx.tmpl) | See the [templates](../internal/configs/version1) | See the [templates](../internal/configs/version1) |
+| Configuration templates *1 | See the [template](https://github.com/kubernetes/ingress-nginx/blob/v1.11.0/rootfs/etc/nginx/template/nginx.tmpl) | See the [templates](../internal/configs/version1) | See the [templates](../internal/configs/version1) |
```

Comment:
different repo

Reply:
yep I noticed too, but that's in the release branch already. I guess we'll have to fix it later.